client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -82,18 +83,23 @@ const EditorSharingAccordion = React.createClass( {
 	},
 
 	hideSharing: function() {
-		return this.props.site &&
-			this.props.site.jetpack &&
-			! this.props.site.isModuleActive( 'publicize' ) &&
-			! this.props.site.isModuleActive( 'sharedaddy' ) &&
-			! this.props.site.isModuleActive( 'likes' );
+		return ( this.props.site &&
+				this.props.site.jetpack &&
+				! this.props.site.isModuleActive( 'publicize' ) &&
+				! this.props.site.isModuleActive( 'sharedaddy' ) &&
+				! this.props.site.isModuleActive( 'likes' ) ) ||
+			( this.props.site &&
+				this.props.site.jetpack &&
+				this.props.postType === 'page' &&
+				! this.props.site.isModuleActive( 'sharedaddy' ) &&
+				! this.props.site.isModuleActive( 'likes' ) );
```

Comment:
Given that we're already using the logic in a few components already, thinking it makes sense for a new selector `isPublicizeEnabled`? Roughly:

```js
export function isPublicizeEnabled( state, siteId, postType ) {
	return (
		false !== isJetpackModuleActive( state, siteId, 'publicize' ) &&
		true !== getSiteOption( state, siteId, 'publicize_permanently_disabled' ) &&
		postTypeSupports( state, siteId, postType, 'publicize' )
	);
}
```

(Or with `postId` argument instead of `postType`, in `state/posts/selectors.js` or `state/ui/editor/selectors.js`)